docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md

```diff
@@ -0,0 +1,243 @@
+
+## Log Types
+
+The MongoDB logs are generated in files as configured in the configuration file /var/log/mongodb/mongodb.log. For more details on MongoDB logs, see [this](https://docs.mongodb.com/manual/reference/log-messages/) link.
+
+## Fields creation in Sumo Logic for MongoDB
+
+Following are the [Fields](https://help.sumologic.com/docs/manage/fields/) which will be created as part of MongoDB App install if not already present.
+
+- **`db.cluster.name`** - User configured. Enter a name to identify this MongoDb cluster. This cluster name will be shown in the Sumo Logic dashboards.
+- **`db.system`** - Has fixed value of mongodb.
+- **`deployment.environment`** - User configured. This is the deployment environment where the Mongodb cluster resides. For example: dev, prod or qa.
+- **`sumo.datasource`** - has a fixed value of mongodb.
+
+## Prerequisites
+
+By default, MongoDB logs are stored in a log file.
+
+1.  Configure logging verbosity in MongoDB : MongoDB logs have six levels of verbosity. All logging settings are located in [MongoDB.conf](https://docs.mongodb.com/manual/reference/method/db.setLogLevel/). To select a level, set loglevel to one of:
+
+-   0 is the MongoDB's default log verbosity level, to include [Informational](https://docs.mongodb.com/manual/reference/log-messages/#std-label-log-severity-levels) messages.
+-   1 to 5 increases the verbosity level to include[ Debug](https://docs.mongodb.com/manual/reference/log-messages/#std-label-log-severity-levels) messages.
+
+2.  Configure MongoDB to log to a Local file :
+
+Configuring MongoDB logs to go to log files. By default, MongoDB logs are stored in `/var/log/mongodb/mongodb.log`. The default directory for log files is listed in the MongoDB.conf file. To configure the log output destination to a log file, use one of the following settings, either in the [configuration file](https://docs.mongodb.com/manual/reference/configuration-options/) or command-line:
+
+-   Configuration file: The [systemLog.destination](https://docs.mongodb.com/manual/reference/configuration-options/#mongodb-setting-systemLog.destination) option for file.
+-   Command-line:
+      -   The [--logpath](https://docs.mongodb.com/manual/reference/program/mongod/#std-option-mongod.--logpath) option for [mongod](https://docs.mongodb.com/manual/reference/program/mongod/#mongodb-binary-bin.mongod) for file.
+      -   The [--logpath](https://docs.mongodb.com/manual/reference/program/mongos/#std-option-mongos.--logpath) option for [mongos](https://docs.mongodb.com/manual/reference/program/mongos/#mongodb-binary-bin.mongos) for file.
+
+## Configure MongoDB Logs Collection
+
+### Step 1: Set up Collector
+
+:::note
+If you want to use an existing OpenTelemetry Collector, you can skip this step by selecting the **Use an existing Collector** option.
+:::
+
+If you want to create a new Collector please select **Add a new Collector** option.
```

Comment:
```suggestion
To create a new Collector:
	
1. Select the **Add a new Collector** option.
2. Select the platform for which you want to install the Sumo OpenTelemetry Collector.
	
This will generate a command which can be executed in the machine which needs to get monitored. Once executed it will install the Sumo Logic OpenTelemetry Collector agent.
```